testnet/stacks-node/src/nakamoto_node/miner.rs

```diff
@@ -115,6 +129,7 @@ impl BlockMinerThread {
             burn_block,
             event_dispatcher: rt.event_dispatcher.clone(),
             parent_tenure_id,
+            miners_stackerdb,
```

Comment:
I'm not sure this needs to be a member of `BlockMinerThread`.  You're basically connecting to yourself -- you can afford to instantiate a `StackerDBSession` each time you mine a block, because there's no gain to be had by preserving the TCP window across multiple block productions.

Reply:
Alright will do :D

Reply:
Done.